zipline/data/resample.py

```diff
@@ -652,7 +649,14 @@ def first_trading_day(self):
         return self._reader.first_trading_day
 
     def get_value(self, sid, dt, field):
-        return self._reader.get_value(sid, dt, field)
+        # Give an empty result if no data is present.
+        try:
+            return self._reader.get_value(sid, dt, field)
+        except NoDataOnDate:
+            if field == 'volume':
+                return 0.0
```

Comment:
Could you return `0` (i.e. an integer) instead, as we do in most other cases where we return a 'no volume' value?